sdks/python/tox.ini

```diff
@@ -33,6 +33,7 @@ extras = test
 whitelist_externals =
   false
   time
+  bash
```

Comment:
If we are running this in windows, does this mean, bash, time etc. needs to be available in the windows installation? Are they available in the github provided image?

Reply:
I added bash to whitelist to get rid of warning messages. It is raised because I had to explicitly use bash in *-win tox testenvs. Bash is available on the github runners:

> When specifying a bash shell on Windows, the bash shell included with Git for Windows is  used.

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell

false and time command are available in mentioned bash shell on windows